config.go

```diff
@@ -379,6 +385,36 @@ func (c *Config) HelmDeploy() {
 		"--install")
 
 	c.runCmd(append([]string{"helm", "upgrade", c.circleBranch(), c.Deploy.ChartName, "-f", envFile}, helmArgs...)...)
+
+	// TODO should exec from output or use something like this https://github.com/kubernetes/client-go/blob/master/examples/out-of-cluster-client-configuration/main.go#L74
+	out, err := exec.Command(LoadBalancerCommand).Output()
+	if err != nil {
+		log.Fatal(LoadBalancerCommand, "failed piping loadbalancer output with error", err.Error())
+	}
+	loadBalancerURL := string(out)
+	api, err := cloudflare.New(c.CloudFlareKey, c.CloudFlareEmail)
+	if err != nil {
+		log.Fatal("cloudflare could not instantiate, failed with error ", err.Error())
+	}
+	// Documentation for record types are found here https://api.cloudflare.com/#dns-records-for-a-zone-create-dns-record
+	// TODO not sure how configurable we want to be with cloudflare zones + dns stuff
+	// TODO also should we configure cloudflare proxy from cli
+	response, err := api.CreateDNSRecord(
```

Comment:
One of the things we need to ensure is that if a domain already exists that it is updated with the new endpoint. Does this do that?